app/CocoaPods/CPANSIEscapeHelper.m

```diff
@@ -0,0 +1,47 @@
+//
+//  CPANSIEscapeHelper.m
+//  CocoaPods
+//
+//  Created by Michael Vilabrera on 9/5/15.
+//  Copyright (c) 2015 CocoaPods. All rights reserved.
+//
+
+#import "CPANSIEscapeHelper.h"
+#import <Fragaria/MGSFragariaFramework.h>
+
+@implementation CPANSIEscapeHelper
+
+- (instancetype) init {
+  self = [super init];
+  if (!self) {
+    return  nil;
+  }
+  
+  
+  NSData *fontData = [[NSUserDefaults standardUserDefaults] valueForKey:MGSFragariaPrefsTextFont];
+  NSFont *font = [NSUnarchiver unarchiveObjectWithData:fontData];
+    
+  self.font = font;
+  
+  self.ansiColors[@(AMR_SGRCodeFgBlack)] = [NSColor colorWithCalibratedRed:0.180f green:0.000f blue:0.008f alpha:1.00f];
```

Comment:
Can you drop the `f` on the end of all these numbers. `NSColor` takes `CGFloat` which is actually a double on most platforms and not a float.